allennlp/modules/contextual_encoder/character_encoder.py

```diff
@@ -0,0 +1,172 @@
+from typing import Sequence, Dict, List
+
+import torch
+import numpy as np
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.modules.highway import Highway
+from allennlp.modules.masked_layer_norm import MaskedLayerNorm
+from allennlp.nn.util import add_sentence_boundary_token_ids
+
+_DEFAULT_FILTERS = ((1, 4), (2, 8), (3, 16), (4, 32), (5, 64))
```

Comment:
Make the filters required input, too much potential for hidden problem with defaults.